dpnp/dpnp_algo/dpnp_algo.pyx

```diff
@@ -255,17 +255,17 @@ Internal functions
 """
 cpdef DPNPFuncType dpnp_dtype_to_DPNPFuncType(dtype):
 
-    if dtype in [numpy.float64, 'float64']:
+    if dtype in [numpy.float64, 'float64', float]:
```

Comment:
type `float` is 32 bit.

Reply:
Python's `float` is 64 bit
```python
a = 2.0
type(a)  # <class 'float'>
np.array(a).dtype  # dtype('float64')
```

Reply:
I'm afraid we will have a `trap` here. It might be interpreter specific. Do you think we really need "python type" mentioned in this code? It would be great to avoid it at this code.